content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -187,14 +187,16 @@ bases:
 ```
 
 ### How to deploy, upgrade and uninstall using the repo
-* Deploy: `kubectl apply -k $YOUR_REPO`
-* Upgrade:
-    1. (Recommended) backup your data storages for KFP.
-    1. Edit `ref={{% pipelines/latest-version %}}` to a version you want to upgrade to.
+Deploy: `kubectl apply -k $YOUR_REPO`
+
+Upgrade:
```

Comment:
The list isn't rendering properly on the page - see preview:
https://deploy-preview-1398--competent-brattain-de2d6d.netlify.com/docs/pipelines/standalone-deployment-gcp/#how-to-deploy-upgrade-and-uninstall-using-the-repo

To fix the problem, add an empty line before the first list item.